base/special/j0f.jl

```diff
@@ -0,0 +1,324 @@
+#     1 - 9 / 128 s^2 + 11025 / 98304 s^4 - ..., where s = 1 / x.
+# For x >= 2, We approximate pzero by
+#     pzero(x) = 1 + R / S
+# where  R = pR0 + pR1 * s^2 + pR2 * s^4 + ... + pR5 * s^10
+#     S = 1 + pS0 * s^2 + ... + pS4 * s^10
+# and
+#     |pzero(x) - 1 - R / S| <= 2^-60.26
+const pR8 = (0.0000000000f00, # 0x00000000
+             -7.0312500000f-02, # 0xbd900000
+             -8.0816707611f+00, # 0xc1014e86
+             -2.5706311035f+02, # 0xc3808814
+             -2.4852163086f+03, # 0xc51b5376
+             -5.2530439453f+03, # 0xc5a4285a
+             )::NTuple{6,Float32}
+const pS8 = (1.1653436279f+02, # 0x42e91198
+             3.8337448730f+03, # 0x456f9beb
+             4.0597855469f+04, # 0x471e95db
+             1.1675296875f+05, # 0x47e4087c
+             4.7627726562f+04, # 0x473a0bba
+             )::NTuple{5,Float32}
+# for x in [8, 4.5454] = 1 / [0.125, 0.22001]
+const pR5 = (-1.1412546255f-11, # 0xad48c58a
+             -7.0312492549f-02, # 0xbd8fffff
+             -4.1596107483f+00, # 0xc0851b88
+             -6.7674766541f+01, # 0xc287597b
+             -3.3123129272f+02, # 0xc3a59d9b
+             -3.4643338013f+02, # 0xc3ad3779
+             )::NTuple{6,Float32}
+const pS5 = (6.0753936768f+01, # 0x42730408
+             1.0512523193f+03, # 0x44836813
+             5.9789707031f+03, # 0x45bad7c4
+             9.6254453125f+03, # 0x461665c8
+             2.4060581055f+03, # 0x451660ee
+             )::NTuple{5,Float32}
+# for x in [4.547, 2.8571] = 1 / [0.2199, 0.35001]
+const pR3 = (-2.5470459075f-09, # 0xb12f081b
+             -7.0311963558f-02, # 0xbd8fffb8
+             -2.4090321064f+00, # 0xc01a2d95
+             -2.1965976715f+01, # 0xc1afba52
+             -5.8079170227f+01, # 0xc2685112
+             -3.1447946548f+01, # 0xc1fb9565
+             )::NTuple{6,Float32}
+const pS3 = (3.5856033325f+01, # 0x420f6c94
+             3.6151397705f+02, # 0x43b4c1ca
+             1.1936077881f+03, # 0x44953373
+             1.1279968262f+03, # 0x448cffe6
+             1.7358093262f+02, # 0x432d94b8
+             )::NTuple{5,Float32}
+# for x in [2.8570, 2] = 1 / [0.3499, 0.5]
+const pR2 = (-8.8753431271f-08, # 0xb3be98b7
+             -7.0303097367f-02, # 0xbd8ffb12
+             -1.4507384300f+00, # 0xbfb9b1cc
+             -7.6356959343f+00, # 0xc0f4579f
+             -1.1193166733f+01, # 0xc1331736
+             -3.2336456776f+00, # 0xc04ef40d
+             )::NTuple{6,Float32}
+const pS2 = (2.2220300674f+01, # 0x41b1c32d
+             1.3620678711f+02, # 0x430834f0
+             2.7047027588f+02, # 0x43873c32
+             1.5387539673f+02, # 0x4319e01a
+             1.4657617569f+01, # 0x416a859a
+             )::NTuple{5,Float32}
+
+# Note: This function is only called for ix>=0x40000000 (see below)
+function pzerof(x::Float32)
+    ix = reinterpret(UInt32, x) & 0x7fffffff
+    # @assert ix >= 0x40000000 && ix <= 0x48000000
+    z = 1f0 / (x * x)
+    if ix >= 0x41000000
+        p = pR8
```

Comment:
Yeah, I'll certainly try that and compare the performance... I haven't done that mainly because I haven't spent any time on this (or similar) code path and I'm curious to see what the performance impact is. (and how the compiler can optimize these.)